taichi/runtime/llvm/llvm_runtime_executor.h

```diff
@@ -85,6 +85,15 @@ class LlvmRuntimeExecutor {
         fetch_result_uint64(i, result_buffer));
```

Comment:
We can finally remove this API afterwards right? `fetch_result_uint64` also crept into `device.h` and should be removed imo

Reply:
Yeah, after the refactoring of all backends completes.